generated/nidmm/library.py

```diff
@@ -575,6 +576,14 @@ def niDMM_close(self, vi):  # noqa: N802
                 self.niDMM_close_cfunc.restype = ViStatus_ctype  # noqa: F405
         return self.niDMM_close_cfunc(vi).value
 
+    def niDMM_error_message(self, vi, error_code, error_message):  # noqa: N802
```

Comment:
if you add this function to NI-DMM, you should remove the one we no longer use.

Reply:
Fixed